core/modules/layout/layout.api.php

```diff
@@ -140,12 +137,11 @@ function hook_layout_context_info() {
  *   - file: Optional. If using theme or preprocess functions, specify the path
  *     to the containing file. This file path is NOT the path to the class.
  *     Class paths and loading is done through hook_autoload_info().
- *   - path: The relative path from this module to the directory containing any
+ *   - template: The relative path from this module to the directory containing any
```

Comment:
`path` is still a valid key. See `layout_theme()` for where it is used. So it should be left as-is. We should add a separate item for `template` though, which isn't currently documented.